pages/events/_slug.vue

```diff
@@ -80,7 +80,7 @@ function getBackgroundUris (background: string): [string, string] {
     }
   }
 })
-export default class extends QiskitPage {
+export default class EventsPage extends QiskitPage {
```

Comment:
I think `EventPage` or `EventEntry` is more appropriate because this component is for a single event's page